tldw_Server_API/tests/Billing/test_overage_enforcement_integration.py

```
@@ -0,0 +1,192 @@
+import pytest
+
+from tldw_Server_API.app.core.Billing import enforcement as enforcement_module
+from tldw_Server_API.app.core.Billing.enforcement import (
+    BillingEnforcer,
+    EnforcementAction,
+    LimitCategory,
+    UsageSummary,
+)
+
+
+@pytest.fixture()
```

Ruff (0.15.6)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)